### DISCUSSION TITLE
NCEI website dead

### DISCUSSION THREAD

**Discussion by VictoriAragon** [earthlab-education/Earth-Analytics-AY24#452] [Q&A]
Been trying to access the NCEI climate data all weekend so I can update my climate station and it times out. Anyone else having this problem?

**Comment by tjstogoski** [comment]
I'm having the same issue. No response from https://www.ncei.noaa.gov, let alone the API.

**Reply by allenmoench** [reply]
Same here! I'll move on to the species assignment and keep checking back to see if they get their site back up.

**Reply by brglea** [reply]
Same! I had finished and just needed to do the portfolio post and it timed out last week and I gave up thinking it was just something wrong with me so it's helpful to see others are experiencing the same issues. Fingers crossed their site goes back up!

**Comment by eculler** [comment]
@earthlab-education/ea-students @earthlab-education/ea-instructors yeah, it looks like their site is down :( We were expecting it to go down sometime last week because they had a notice about it, but it seems to be happening now instead.

Let's give it a few days and check again -- go ahead and work on the migration assignment instead. We can check back in later in the semester. If their site continues to have outages I'll remove the climate portfolio assignment.

**Comment by cmq879** [comment]
The NCEI website is back up finally, as of late last week! 
https://www.ncei.noaa.gov/access
